python/lsst/pipe/base/pipeline.py

```diff
@@ -484,6 +526,9 @@ def _addConfigImpl(self, label: str, newConfig: pipelineIR.ConfigIR):
     def toFile(self, filename: str):
         self._pipelineIR.to_file(filename)
 
+    def toUri(self, uri: str):
+        self._pipelineIR.to_uri(uri)
```

Comment:
Is this supposed to return something?  (Same for the preexisting `toFile`.)

If not, is it too late to rename these to something that more strongly suggests an in-place operation.

Finally, MyPy at least is going to complain if you don't add `-> None` to methods that don't return anything while annotating the arguments.

Reply:
I'm happy to change the new method name, but technically I think I would have to go through a process to change the other, which does not seem worth it as at some point I would say we should just drop the old one.